starter_code/views/partials/randomBeerPartial.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="random-card">
+  <img src='{{image_url}}'>
+  <div class="text-random">
+    <h1>{{name}}</h1>
+    <h2>{{tagline}}</h2>
+    <p>{{description}}</p>
+    <p><b>Food Pairing</b></p>
+    <ul>
+      {{#each food_pairing}}
+        <li>{{this}}</li>
+      {{/each}}
+    </ul>
+    <p><b>Brewers Tips:</b> {{brewers_tips}}</p>
+  </div>
+</div>
```

Comment:
tu peux rappeler le this du beers.hbs : 
h1>{{this.name}}</h1>
    <h2>{{this.tagline}}</h2>
    <p>{{this.description}}</p>